ggshield/cmd/sca/scan.py

```diff
@@ -41,6 +58,7 @@ def scan_diff_cmd(
 
 @scan_group.command(name="all")
```

Comment:
We need to add these lines so that we can use the `ggshield sca scan all` command

```
@click.argument(
    "directory",
    type=click.Path(exists=True, readable=True, path_type=Path, file_okay=False),
    required=False,
)
```